static/css/inbox.less

```diff
@@ -870,15 +870,61 @@ a.fa:hover {
         color: #000000;
       }
     }
+
+    .sub-actions {
+      background-color: #FFFFFF;
+      height: 56px;
+      padding: 0;
+      margin: 0 0 1px 0;
+      border-radius: 0;
+      border: none;
+      display: flex;
+      box-shadow: 0 -6px 12px rgba(0, 0, 0, 0.4);
+
+      .mm-icon-caption {
+        border: none;
+        line-height: 1;
+        color: @label-color;
+        max-width: none;
+        margin: 6px 0 0;
+        .reports-icon {
+          display: block;
+          margin: 0 auto 6px auto;
+          svg {
+            width: 24px;
+            height: 24px;
+            path {
+              fill: @deceased-color;
```

Comment:
I'm not sure about using the `@deceased-color` here because it makes the code harder to read - has the option died?

Can you use label-color? Or can we rename deceased-color to cover both use cases?

Reply:
I've renamed `@deceased-color` to `@inactive-color`. `@label-color` was too dark, while `@deceased-color` was truly very close to the color requested by the design team.